samples/grpc/local-drone-control-java/src/main/java/local/drones/Drone.java

```diff
@@ -40,6 +40,7 @@ public static final class ReportPosition implements Command {
     public final Position position;
     public final ActorRef<Done> replyTo;
 
+    @JsonCreator
```

Comment:
not important, but this should only be needed for single arg constructors

Reply:
Ah, didn't realise. Just sprinkled when there was a failure for one of the messages. Have updated.

When retesting, discovered that some cross-cluster paths had not been exercised. Another missing `CborSerializable` and some native image reflect config needed. So that was useful.